staff/ana-sanjuan/app-template/login.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Register</title>
+</head>
+<body>
+    <header>
+        <h1>Login</h1>
+        <nav>
+            <a href="home.html">Home</a> |
+            <a href="news.html">News</a> |
+            <a href="about.html">About</a> |
+            <a href="contact.html">Contact</a>|
+            <a href="register.html">Register</a> |
+            <a href="login.html">Login</a> |
+            <a href="profile.html">Profile</a> |
+        </nav>
+    </header>
+    <hr>
+    <article>
```

Comment:
why `article` wrapping a form here? it doesnt seem to have sense, please RTFM

Reply:
I have deleted it here. thanks!